lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -582,10 +582,13 @@ module.exports = class MethodMappings {
 
           return value;
         },
-
+        
         async clearElementValue(webElementOrId) {
           const element = this.getWebElement(webElementOrId);
           await element.clear();
+          var value = await element.getAttribute('value');
+          const backArr = Array(value.length).fill(Key.BACK_SPACE);
```

Comment:
We should only be sending the back spaces if a value exists (`value` should be a string but not empty).